synapse/tests/test_datamodel.py

```diff
@@ -245,8 +257,17 @@ def test_datamodel_getPropInfo(self):
 
         model.addTufoForm('foo')
         model.addTufoProp('foo', 'meow', ptype='foo:baz')
+        model.addTufoProp('foo', 'bark', doc='lala')
 
+        self.eq(model.getPropInfo('foo:meow', 'req'), False)
+        self.eq(model.getPropInfo('foo:meow', 'base'), 'meow')
+        self.eq(model.getPropInfo('foo:meow', 'defval'), None)
+        self.eq(model.getPropInfo('foo:meow', 'title'), '')
         self.eq(model.getPropInfo('foo:meow', 'doc'), 'foo bar doc')
+
+        self.eq(model.getPropInfo('foo:bark', 'doc'), 'lala')
```

Comment:
should we add the rest of the checks we perform on `meow`?

Reply:
yup - updating

Reply:
fixed